### DISCUSSION TITLE
Ignore IPV6 and docker subnets

### DISCUSSION THREAD

**Discussion by NCVito** [netalertx/NetAlertX#717] [Q&A]
Is there a way to have it not scan/show IPV6 results and/or internal docker subnets (i.e. 172.x.x.x)?

**Comment by jokob-sk** [comment]
Hi there,

You should be able to ignore devices as per this guide https://github.com/jokob-sk/NetAlertX/blob/main/docs/NOTIFICATIONS.md#ignoring-devices-
